### ISSUE TITLE
IRC: Report user permissions during user list message

### ISSUE THREAD

**Issue by silverbucket** [sockethub/sockethub#546]

**Comment by silverbucket** [comment]
@gregkare didn't we do this? or was there still something left to do?

**Comment by silverbucket** [comment]
@galfert @gregkare @raucao - I forgot what this issue was about (should have added a description). I have a feeling it was related to the work Greg and I did several years back when we had the Berlin meetup. In which case, I think it may have been resolved. Can anyone corroborate or clarify?

**Comment by galfert** [comment]
I guess this issue was about the user roles of channel members. I just checked and it's included in the presence update object:

```json
{
    "@type": "update",
    "actor": {
        "@type": "person",
        "@id": "thedude@irc.libera.chat",
        "displayName": "thedude"
    },
    "target": {
        "@type": "room",
        "@id": "irc.libera.chat/#kosmos-random",
        "displayName": "#kosmos-random"
    },
    "object": {
        "@type": "presence",
        "role": "member"
    },
    "published": "1631010832002",
    "context": "irc"
}
```

So I think this issue can be closed.
